### PR TITLE
Add default preprocessor, add check for existence of valid data

### PR DESCRIPTION
**Pull request by phirework**
* Seems like we've added a whole bunch of languages since the last dataset release and instead of copying the same boilerplate file 20 times and keeping track of new languages I added a default preprocessor that just spits back the sentence. Also deleted all the language preprocessors that were just the boilerplate - let me know if there was a specific reason these were in separate files. Also amended documentation to explain this.
* We have 108 clips from `vot` locale (Votic) where none of it has any votes (because there are 20 living speakers in the world), which was breaking `_calculate_data_set_sizes` since the loop never runs and `train_size` et al are never instantiated, so I added a length check.
* Added an exception handler to skip Attribute errors to deal with the case of a Corpus not having buckets as per above.

I am not a python dev, so please do let me know if there's a more pythonic way of handling any of the above.

### CODE REVIEW

**Comment by johngian** [comment]
Hi @phirework ! Here are some comments for the PR after your feedback request:

* You are deleting some preprocessors (eg. `br.py`) completely but under `preprocessors/__init__.py` still imports them which might raise an error.
* Is there a way I can test it locally? I am not really familiar with the tool (I only run it once with `en` dataset). 
* There are some comments in-line. Might be a little nit-picky but I don't really know the context of the tool. Feel free to omit.

File: src/corporacreator/corpus.py [hunk 1]
```diff
@@ -79,30 +82,32 @@ def _post_process_valid_data(self):
...
-            elif len(dev) + len(validated[validated["continous_client_index"] == i]) <= dev_size:
-                dev = pd.concat([dev, validated[validated["continous_client_index"] == i]])
-            else:
-                train = pd.concat([train, validated[validated["continous_client_index"] == i]])
-
-        self.train = train.drop(columns="continous_client_index")
-        self.dev = dev.drop(columns="continous_client_index")
-        self.test = test[:train_size].drop(columns="continous_client_index")
+
+        if (len(validated) > 0):
```
[the remark this answers is not in the document]
Comment (phirework) [reply src/corporacreator/corpus.py]: Everything wrapped by this length check assumes that `validated` has at least one entry in it, starting with `_calculate_data_set_sizes()`. Skipping the whole block if length = 0 seemed like the least disruptive way of handling it.

I suppose we could instantiate `self.train`, `self.dev`, and `self.test` as empty data frames before this length check...thoughts?

File: src/corporacreator/corpus.py [hunk 2]
```diff
@@ -132,7 +137,10 @@ def save(self, directory):
 
     def _save(self, directory, dataset):
         path = os.path.join(directory, dataset + ".tsv")
-        dataframe = getattr(self, dataset)
-        dataframe.to_csv(
-            path, sep="\t", header=True, index=False, encoding="utf-8", escapechar='\\', quoting=csv.QUOTE_NONE
-        )
+        try:
+            dataframe = getattr(self, dataset)
```
[the remark this answers is not in the document]
Comment (phirework) [reply src/corporacreator/corpus.py]: Hmm which line would you move outside the `try/except`? If the `getattr()` fails the dataframe is undefined and there's nothing to write to CSV. 

Re: logging the error, I assumed that there would be some value in seeing which corpora had empty buckets. If we instantiate self.train/dev/test as empty dataframes then this piece wouldn't be necessary.

**Comment by phirework** [comment]
Thanks for the review! Removed unused preprocessors. 

There's no easy way to test locally, you could generate a live `clips.tsv` using the bundler (https://github.com/Common-Voice/common-voice-bundler/) and then running the result of that through this. I can also send you the full `clips.tsv` file I was working off of for the 2019 H2 corpus. 

Good call on the lint issues, I'm going to file a separate ticket to clean that up.

File: src/corporacreator/corpus.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (johngian) [reply src/corporacreator/corpus.py]: I would prefer that one.

File: src/corporacreator/corpus.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (johngian) [reply src/corporacreator/corpus.py]: > If we instantiate self.train/dev/test as empty dataframes then this piece wouldn't be necessary.
This feels a little bit more pythonic

**Pull request merged**